lemur/plugins/lemur_acme/plugin.py

```diff
@@ -447,6 +471,13 @@ class ACMEIssuerPlugin(IssuerPlugin):
             "validation": "/^-----BEGIN CERTIFICATE-----/",
             "helpMessage": "Certificate to use",
         },
+        {
+            "name": "store_account",
+            "type": "bool",
+            "required": False,
+            "helpMessage": "Disable to create a new account for each ACME request",
+            "default": True,
```

Comment:
Shouldn't the default be `False`? 

```
            # if store_account is checked, add the private_key and registration resources to the options
            if options['store_account']:
```

Reply:
You're right. Having the default be `False` leaves the behaviour as is, otherwise we would change the default behaviour.